From e14f9d17af2b899de3fc2a791e93f717f2b8d5b0 Mon Sep 17 00:00:00 2001
From: Kairui Song <kasong@redhat.com>
Date: Tue, 26 Feb 2019 16:26:51 +0800
Subject: [PATCH] Refine document about IRQBALANCE_BANNED_CPUS

There is no declaration about how irqbalance deal with isolated CPUs or
adaptive-ticks CPUs, and how IRQBALANCE_BANNED_CPUS will override the
behavior of auto-banning of such CPUs. Refine the document to avoid
confusion.

Signed-off-by: Kairui Song <kasong@redhat.com>
---
 irqbalance.1 | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/irqbalance.1 b/irqbalance.1
index 7e508d7..2763c29 100644
--- a/irqbalance.1
+++ b/irqbalance.1
@@ -138,6 +138,8 @@
 would prevent irqbalance from assigning irqs to the 7th-12th cpus (cpu6-cpu11) or
 ‘export IRQBALANCE_BANNED_CPUS=ff000000,00000001‘ would prevent irqbalance from
 assigning irqs to the 1st (cpu0) and 57th-64th cpus (cpu56-cpu63).
+If not specified, irqbalance use mask of isolated and adaptive-ticks CPUs on the
+system as the default value.
 
 .SH "SIGNALS"
 .TP
-- 
2.20.1

